<launch>

  <node name="io_control" pkg="io_control" type="io_control" output="screen" respawn="false">
  <param name="port"       value="/dev/ttyUSB0"/>  
  <param name="baudrate"   value="9600"/>
  <param name="debug_imu"   value="true"/>
  </node>


</launch>



